### DISCUSSION TITLE
gitsigns插件gs.preview_hunk打开的窗口如何进入

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#114] [Q&A]
![image](https://user-images.githubusercontent.com/43649186/188821374-f1cab60a-f652-44b8-92e6-9f5bdfd3e15c.png)
这个窗口如何进入，目前我只能通过鼠标点击 然后才能进入到该窗口

**Comment by nshen** [comment]
我现在没用gitsigns，通常同一个命令调用两次就会进入窗口，打开窗口的快捷键敲两次试试
